build/media_source/templates/administrator/atum/scss/blocks/_form.scss

```
@@ -176,8 +176,12 @@ div.subform-repeatable-group {
         top: 50%;
         right: 100%;
         margin-top: -27px;
-        line-height: 52px;
         border-radius: $border-radius 0 0 $border-radius;
+        padding: 0;
+
+        span {
+          padding: 1.5rem 0.5rem;
+        }
```

```suggestion
        padding: 0;
        border-radius: $border-radius 0 0 $border-radius;

        span {
          padding: 1.5rem .5rem;
        }
```
Fix SCSS code style errors reported by Drone here: https://ci.joomla.org/joomla/joomla-cms/60559/1/25